ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -478,10 +479,10 @@ protected void registerWebSocketEndpoints(final ServerContainer container) {
           ErrorMessages.class
       ));
 
-      final RoutingFilters routingFilters = initializeRoutingFilters(
-          ksqlConfigNoPort, heartbeatAgent);
+      final RoutingFilterFactory routingFilterFactory = initializeRoutingFilters(
```

Comment:
rename to `initializeRoutingFilterFactory`?

Reply:
Done